main.go

```diff
@@ -241,8 +241,8 @@ func main() {
 			},
 		},
 		{
-			Name:    "recent-builds",
-			Aliases: []string{"recent"},
+			Name:    "recent",
+			Aliases: []string{"r"},
```

Comment:
Should we leave the existing subcommand and add a new one for showing recent workflows?

Reply:
We can create a new command for sure.